IMPLEMENTATIONS.md

```diff
@@ -20,6 +20,9 @@ These tools are not endorsed by the OAI.
 | oas3-remote-refs | [github//OverSpeedIO/oas3-remote-refs](https://github.com/OverSpeedIO/oas3-remote-refs) | Node.js | Tool to pull remote refernences and merge them into the definitions of the provided OpenAPI3 speicfication.
 | go-openapi | [github/nasa9084/go-openapi](https://github.com/nasa9084/go-openapi) | Go | Golang struct model for OpenAPI 3.x. |
 | openapi | [github/wzshiming/openapi](https://github.com/wzshiming/openapi) | Go | OpenAPI 3 Specification for golang |
+| open api spex | [/mbuhot/open_api_spex(https://github.com/mbuhot/open_api_spex)] | Elixir | Leverage Open Api Specification 3 (swagger) to document, test, validate and explore your Plug and Phoenix APIs. |
```

Comment:
"Open Api" should be "OpenAPI", and as OAS 3 was never known as 'swagger', unlike OAS 2.0, this should probably be removed or replaced with "fka Swagger".

Reply:
The title of the tool still says "open api spex"